scripts/ai/jobs/AIJobBaleFinderCp.lua

```diff
@@ -0,0 +1,65 @@
+--- Bale finder job.
+---@class AIJobBaleFinderCp : AIJobFieldWorkCp
+AIJobBaleFinderCp = {
```

Comment:
At some point we should really decide if we want everything to prefix with Cp or postfix...